src/components/Search/styles.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/main.scss';
+.search-container {
+    float: right;
+    height: 100vh;
+    overflow: auto;
+    background-color: white;
+    @include flex-center();
+    @include respond-above(md) {
+      width: rem(1140px);
```

Comment:
Reminder, this should change after a rebase from `feature/product-list-ui` 😎 (Close this comment when it changes)